### DISCUSSION TITLE
Using this plugin for just lost links in _resources

### DISCUSSION THREAD

**Discussion by antonvons** [Vinzent03/find-unlinked-files#46] [Q&A]
Will this app work for finding just lost links in _resources folder.

It seems to allow excluding but can it include one folder? 

Thank you.

**Comment by Vinzent03** [comment]
There is a toggle above the directories text area to switch between allow and disallow list. In your case, please disable the toggle to only search within `_resources`.

**Reply by antonvons** [reply]
Thank you, I will give this a try soon.
